schema.sql

```diff
@@ -62,3 +62,7 @@ CREATE TABLE visits (
     FOREIGN KEY (id_vet) REFERENCES vets (id),
     FOREIGN KEY (id_animals) REFERENCES animals (id)
 );
+
+-- Performance audit project
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
```

Comment:
Hi Team 👋🏾 , as was requested in the requirement you are expected  to add the following to your schema file 
`` 
  CREATE INDEX ON visits (animal_id);
   CREATE INDEX ON visits (vet_id);
   CREATE INDEX ON owners (email);
``
add to your database cause it will serve as a basic solution to your performance problems add them, update the PR description with the new screenshots and you are good to go.